sites/public/pages/applications/review/terms.tsx

```diff
@@ -63,6 +64,7 @@ export default () => {
           .then(() => window.scrollTo(0, 0))
       })
       .catch((err) => {
+        setSubmitting(false)
```

Comment:
Curious why we only set submitting to false in case of an error, and not after a successful submit? Is it just because we change routes?